### PR TITLE
Add ability to easily run prettier from CLI

### PR DESCRIPTION
**Pull request by jtomasek**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
     "test-suite": "ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/protractor integration-tests/protractor.conf.ts",
     "debug-test-suite": "TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' node -r ts-node/register --inspect-brk ./node_modules/.bin/protractor integration-tests/protractor.conf.ts",
     "analyze": "NODE_ENV=production NODE_OPTIONS=--max-old-space-size=4096 ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode=production --profile --json | awk '{if(NR>2)print}' > public/dist/stats.json && ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack-bundle-analyzer --mode static -r public/dist/report.html public/dist/stats.json",
-    "plugin-stats": "node ./plugin-stats.js"
+    "plugin-stats": "node ./plugin-stats.js",
+    "prettier": "prettier '**/*.{js,jsx,ts,tsx}'"
```
Comment (benjaminapetersen) [inline frontend/package.json]: Seems handy to me.

**Comment by christianvogt** [comment]
Doesn't hurt anyone to have it. But prettier is run already with eslint. Although running just prettier I'm sure is faster.

Tested it in our repo. Running this command just output a lot of noise to the console. Not sure what you intended to it to do. It's not very useful like this.

Useful options are `--check` and `--write`.

**Comment by jtomasek** [comment]
> Tested it in our repo. Running this command just output a lot of noise to the console. Not sure what you intended to it to do. It's not very useful like this.
My goal was to be able to easily do prettier formatting from CLI. I've thought that `--write` option was the default behaviour. Fixing...

File: frontend/package.json [hunk 2]
```diff
@@ -24,7 +24,8 @@
     "test-suite": "ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/protractor integration-tests/protractor.conf.ts",
     "debug-test-suite": "TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' node -r ts-node/register --inspect-brk ./node_modules/.bin/protractor integration-tests/protractor.conf.ts",
     "analyze": "NODE_ENV=production NODE_OPTIONS=--max-old-space-size=4096 ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode=production --profile --json | awk '{if(NR>2)print}' > public/dist/stats.json && ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack-bundle-analyzer --mode static -r public/dist/report.html public/dist/stats.json",
-    "plugin-stats": "node ./plugin-stats.js"
+    "plugin-stats": "node ./plugin-stats.js",
+    "prettier": "prettier --write '**/*.{js,jsx,ts,tsx}'"
```
Comment (jtomasek) [inline frontend/package.json]: Do we want to also format `json`? Doing so currently affects 23 files.

Comment (christianvogt) [reply frontend/package.json]: We should.
Can we update eslint to run against json files as well so it's enforced by our ci?
Then we fix them all up and can include json files w/ prettier here.

Comment (jtomasek) [reply frontend/package.json]: I've added eslint-plugin-json for json linting and created `json.js` in our `eslint-plugin-console' package.

**Comment by christianvogt** [comment]
Looking good @jtomasek 
Anything else you're still working on?

**Comment by jtomasek** [comment]
> Looking good @jtomasek
> Anything else you're still working on?

I think it's complete now.

**Comment by christianvogt** [comment]
@jtomasek I ran it and since it does not follow `eslint-disable-next-line`, I got a fix applied to `create-bs.tsx`
https://github.com/jtomasek/console/blob/a3a15d501c20e53719532ac9311962bbf46c04e0/frontend/packages/noobaa-storage-plugin/src/components/create-backingstore-page/create-bs.tsx#L559

I suggest you remove this line and run prettier again to commit all changes so that the next user doesn't find any changes when they run it.

**Comment by jtomasek** [comment]
I've removed the `eslint-disable-next-line` comment, used dot notation to stop eslint complaining and added type to make typescript happy.
+ Rebased.

**Comment by spadgett** [comment]
This actually seems to have broken prettier on the CLI for me. I used to be able to do

```
$ yarn run prettier --write public/components/edit-yaml.jsx
```

Now it runs on all files always. Can we rename this to something like `prettier-all`?

**Pull request merged**